From f91d9bfa17478442902517e56199c796ae57efd6 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Sat, 7 Mar 2015 19:58:56 -0800
Subject: [PATCH 8/9] Define __STDC_FORMAT_MACROS to bring in format macros
 when including <inttypes.h> in C++ code

Fixes regression from: 1f22bd54e8565959c1d2c3ed7384cb77e18d0807

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
CC: Dan Albert <danalbert@google.com>
(cherry picked from commit e02479718191c6aa63a23b2a5029a69906ba3d76)
---
 src/DwarfParser.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/DwarfParser.hpp b/src/DwarfParser.hpp
index f6ef738..0c24dc6 100644
--- a/src/DwarfParser.hpp
+++ b/src/DwarfParser.hpp
@@ -13,6 +13,7 @@
 #ifndef __DWARF_PARSER_HPP__
 #define __DWARF_PARSER_HPP__
 
+#define __STDC_FORMAT_MACROS 1
 #include <inttypes.h>
 #include <stdint.h>
 #include <stdio.h>
-- 
2.6.2

